react-native/index.d.ts

```diff
@@ -2563,8 +2563,18 @@ declare module "react" {
         ref?: Ref<DrawerLayoutAndroidStatic & ViewStatic>
     }
 
+    interface DrawerPosition {
```

Comment:
I might be wrong, but I don't think this is the right place to add it. `DrawerLayoutAndroidProperties` already defines a [`drawerPosition`](https://github.com/victor95pc/DefinitelyTyped/blob/fcc78898d3baf2e1ed7c45f693fdc4e2aaf94530/react-native/index.d.ts#L2505) property. It's type should be changed from `any` to [`DrawerConsts`](https://github.com/facebook/react-native/blob/3683beb88ab3233038ac32b9c33112d537478088/Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js#L23). Probably need to add `DrawerConsts` first.